static/src/stylesheets/module/commercial/_adverts-capi.scss

```diff
@@ -173,6 +181,11 @@
     background: $paid-article-subheader;
     border-top-color: $paid-article-brand;
 
+    &:hover,
+    &:focus {
+        background: darken($paid-article-subheader, 7%);
```

Comment:
My concern with this darken + whatever magic number pattern is that it will get messy pretty soon. I've used it on one project and it reaaaaaaallly fast became totally random percentages across all components. Based on the current mood basically.